action.yml

```diff
@@ -85,6 +104,16 @@ runs:
           venv/bin/python direct_get.py --overwrite --version="${{ inputs.console_version }}" --target_directory=solsta_console/${{ inputs.console_version }}/ --console_credentials=client_credentials.json --component=console
         fi
 
+    - name: Download Console Tools (Windows)
```

Comment:
Why have two versions of this step?

Reply:
Ah, Mr. Nathan -- you must look very closely to see the difference.  The Python path differs.
You see: When you do python -m to create a new venv, it puts the python binary in a different directory depending on whether the host is Windows or Linux/Mac.  In the former, it goes in Scripts/ and in the later, bin/.
Crazy, huh?  The pip step is also duplicated in there for the same reason.

Reply:
I think there are probably ways around it:
```yaml
- name: Set Python venv path (Linux/macOS)
  if: runner.os != 'Windows'
  run: echo "VENV_PATH=venv/bin" >> $GITHUB_ENV

- name: Set Python venv path (Windows)
  if: runner.os == 'Windows'
  run: echo "VENV_PATH=venv/Scripts" >> $GITHUB_ENV
```
Then you could use it in your actions below it like so:
```yaml
- name: Download Console Tools
  shell: bash
  working-directory: solsta_work
  run: |
     ${{ env.VENV_PATH }}/python direct_get.py \
       --overwrite \
       --version="${{ inputs.console_version }}" \
       --target_directory=solsta_console/${{ inputs.console_version }}/ \
       --console_credentials=client_credentials.json \
       --component=console
```
What do you think?

Reply:
Done